cristin-person/src/main/java/no/unit/nva/cristin/person/model/nva/Award.java

```diff
@@ -0,0 +1,24 @@
+package no.unit.nva.cristin.person.model.nva;
+
+import static no.unit.nva.cristin.model.JsonPropertyNames.AFFILIATION;
+import static no.unit.nva.cristin.model.JsonPropertyNames.DISTRIBUTION;
+import static no.unit.nva.cristin.model.JsonPropertyNames.TITLE;
+import static no.unit.nva.cristin.model.JsonPropertyNames.TYPE;
+import static no.unit.nva.cristin.model.JsonPropertyNames.YEAR;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import no.unit.nva.commons.json.JsonSerializable;
+import no.unit.nva.model.Organization;
+import no.unit.nva.model.TypedLabel;
+
+public record Award(@JsonProperty(TITLE) String title,
```

Comment:
Title? Name? Am I right in remembering we use 'name' elsewhere?

Reply:
i can update to name?

Reply:
Award name, yes

Reply:
Done